core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -1069,6 +1079,7 @@ class ZKCheckedEphemeral(path: String,
     val prefix = path.substring(0, index)
     val suffix = path.substring(index, path.length)
     debug("Path: %s, Prefix: %s, Suffix: %s".format(path, prefix, suffix))
+    info(s"Creating $path in $isSecure mode")
```

Comment:
isSecure is a boolean and we need to translate that to the corresponding text in the logging message.